### ISSUE TITLE
What's your preferred documentation format?

### ISSUE THREAD

**Issue by levitte** [buildbot/buildbot_travis#110]
Hi, I've started to use bbtravis for OpenSSL's upcoming buildbot, and am interested enough to provide documentation (the documentation is currently appalling) as time permits.  To do it right, I would like to know what documentation format you'd prefer, @tardyp

**Comment by srggrs** [comment]
Hi where I can find some documentation on how to use it from the UI? Thank you!

**Comment by tardyp** [comment]
Hello, As of now, the documentation is in the form of the Readme.md at the top of the project.

https://github.com/buildbot/buildbot_travis/blob/master/README.rst

**Comment by tardyp** [comment]
there is also the buildbot documentation for the basics. I guess neither of them have a full documentation on the UI beyond the basic tutorials.

**Comment by srggrs** [comment]
Yeah exactly also I found bugs such as #116 and would like to fix but I had issues in setting up the development env (and I am not the only one #114). Would be great having some CONTRIBUTING guidelines and instructions.

Do I need to install in editable mode buildbot first or first buildbot_travis and then buildbot. Some help would be appreciated :)
thank you!

**Comment by tardyp** [comment]
any help is greatly appreciated. Indeed, you should first install buildbot in development mode.
https://docs.buildbot.net/latest/developer/quickstart.html

and then buildbot_travis in that same environment.
